src/command/output.rs

```diff
@@ -103,7 +103,7 @@ impl RoverStdout {
                 }
             }
             RoverStdout::Introspection(introspection_response) => {
-                eprintln!("Introspection Response:");
+                eprintln!("Introspection Response:\n");
```

Comment:
why the extra newline?

Reply:
For readability -- I couldn't visually distinguish where introspection response started. Happy to remove tho!